utemplates_apply.php

```diff
@@ -73,9 +73,9 @@
 // Begin of: manage form submission.
 $utemplateman->formdata = $applyutemplate->get_data();
 if ($utemplateman->formdata || ($confirm == SURVEYPRO_CONFIRMED_YES)) {
-    // Here I don't need to execute validate_xml because xml was validated at upload time
-    // Here I only need to verfy that plugin versions still match
-    $utemplateman->check_items_versions();
+    // Here I don't need to execute validate_xml because xml was validated at upload time.
+    // Here I only need to verfy that plugin versions still match.
+    // $utemplateman->check_items_versions();
```

Comment:
This is only an example. I've seen that you here and there use to keep old code commented. One line or multiple lines. That's not good in general. If you've to explain something do it, but not keeping old code. In fact I bet the codechecker will, probably cry about it.